IndicoV2.V1Adapters/Models/V1ModelClientAdapter.cs

```diff
@@ -33,5 +34,8 @@ public class V1ModelClientAdapter : IModelClient
                 .Data(data)
                 .Exec(cancellationToken))
             .Id;
+
```

Comment:
if you're ready to convert this to the StrawberryShake graphql call, it makes sense to get it done while you're in here.